package.json

```diff
@@ -64,23 +64,23 @@
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-config-prettier": "^8.6.0",
     "eslint-plugin-import": "^2.27.5",
-    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-prettier": "^5.0.0",
     "isomorphic-fetch": "^3.0.0",
     "mocha": "^10.0.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
-    "prettier": "^2.8.3",
+    "prettier": "^3.0.0",
     "sinon": "^15.0.1",
     "standard-version": "^9.5.0",
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5",
-    "web3": "^1.8.2"
+    "web3": "^1.10.0"
   },
   "dependencies": {
     "bignumber.js": "^9.1.1",
     "ethereumjs-util": "^7.1.5",
-    "web3-eth-abi": "^1.8.2",
-    "web3-providers-http": "^1.8.0",
-    "web3-utils": "^1.8.2"
+    "web3-eth-abi": "^1.10.0",
+    "web3-providers-http": "^1.10.0",
```

Comment:
Nice! There were some improvements in this release

https://github.com/web3/web3.js/releases/tag/v1.10.0


![image](https://github.com/ERC725Alliance/erc725.js/assets/31145285/43c25ac5-d990-4b29-95c3-9534a2535d5c)